### PR TITLE
Pricing Section Added And Phone Number Updated

### PR DESCRIPTION
**Pull request by stevenixon1999**
Pricing Section Added 
New car models added
 Phone Number Updated

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -10,4 +11,28 @@ document.addEventListener('DOMContentLoaded', function () {
     const pickDate = pickDateInput.value;
     dropDateInput.setAttribute('min', pickDate);
   });
+
+  function initializeAccordion() {
+    const accordionBtns = document.getElementsByClassName("accordion-button");
```
Comment (vaishak101) [inline assets/js/index.js]: Use querySelector

File: our-pricing/index.html [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+          <li><a href="tel:+91 7710087056">Enquiry & Booking : +91 7710087056</a> / <a href="tel:+91 7710087057">+91 7710087057</a></li>
+        </ul>
+      </div>
+    </div>
+    <p class="copyright">Copyright © 2024 | Alphy tours and travels</p>
+  </footer>
+
+  <script>
+
+    document.addEventListener('DOMContentLoaded', function () {
```
Comment (vaishak101) [inline our-pricing/index.html]: Create a seperate js yo use it on home and out pricing

**Review by vaishak101 — CHANGES_REQUESTED** [review]

File: assets/js/index.js (hunk 1 above)
Comment (stevenixon1999) [reply assets/js/index.js]: Changed it to querySelectorAll

File: our-pricing/index.html (hunk 2 above)
Comment (stevenixon1999) [reply our-pricing/index.html]: Created a Separate file for accordion js

**Pull request merged**